releases/release-5.1.1.md

```diff
@@ -0,0 +1,113 @@
+---
+title: TiDB 5.1.1 Release Notes
+---
+
+# TiDB 5.1.1 Release Notes
+
+Release Date: July 30, 2021
+
+TiDB version: 5.1.1
+
+## Compatibility change
+
++ TiDB
+
+    - For users upgrading from TiDB 4.0, the value of tidb_multi_statement_mode is now OFF. It is recommended to use the multi-statement feature of your client library instead, see the documentation on tidb_multi_statement_mode for additional details. [#25751](https://github.com/pingcap/tidb/pull/25751)
+
```

Comment:
- exec: access the table_storage_stats need super privilege [#26352](https://github.com/pingcap/tidb/pull/26352)
    - Accessing information_schema.user_privileges will now requires the SELECT privilege on mysql.user in order to show other user's privileges. [#26311](https://github.com/pingcap/tidb/pull/26311)
    - Reading from the table information_schema.cluster_hardware now requires the CONFIG privilege.
    - Reading from the table information_schema.cluster_info now requires the Process privilege.
    - Reading from the table information_schema.cluster_load now requires the Process privilege.
    - Reading from the table information_schema.cluster_systeminfo now requires the Process privilege.
    - Reading from the table information_schema.cluster_log now requires the Process privilege. [#26297](https://github.com/pingcap/tidb/pull/26297)   
    - Reading from the table information_schema.cluster_config now requires the CONFIG privilege. [#26150](https://github.com/pingcap/tidb/pull/26150)
    - Improve the MySQL compatibility of str_to_date for %b/%M/%r/%T [#25768](https://github.com/pingcap/tidb/pull/25768)    - exec: access the table_storage_stats need super privilege [#26352](https://github.com/pingcap/tidb/pull/26352)
    - Accessing information_schema.user_privileges will now requires the SELECT privilege on mysql.user in order to show other user's privileges. [#26311](https://github.com/pingcap/tidb/pull/26311)
    - Reading from the table information_schema.cluster_hardware now requires the CONFIG privilege.
    - Reading from the table information_schema.cluster_info now requires the Process privilege.
    - Reading from the table information_schema.cluster_load now requires the Process privilege.
    - Reading from the table information_schema.cluster_systeminfo now requires the Process privilege.
    - Reading from the table information_schema.cluster_log now requires the Process privilege. [#26297](https://github.com/pingcap/tidb/pull/26297)   
    - Reading from the table information_schema.cluster_config now requires the CONFIG privilege. [#26150](https://github.com/pingcap/tidb/pull/26150)
    - Improve the MySQL compatibility of str_to_date for %b/%M/%r/%T [#25768](https://github.com/pingcap/tidb/pull/25768)